cms/static/sass/elements/_modules.scss

```diff
@@ -131,6 +131,7 @@
         height: ($baseline*5);
         margin-bottom: ($baseline/2);
         border: 1px solid $green-d2;
+        padding: 0;
```

Comment:
Let's move padding up a line, beneath `margin-bottom`. We prefer to follow [Concentric CSS ordering](https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#property-and-declaration-order) where we define styles going from outside to inside.